.github/workflows/label-needs-triage.yml

```diff
@@ -0,0 +1,20 @@
+name: Add needs-triage label to new issue
```

Comment:
Should we remove the Fabric Bot rule?   Probably no harm in having two things apply the label... I hope?

Reply:
I think this rule is straight forward and I wanted to use this one as an experiment.

If you don't have objections, I will keep both of them so we are consistent with other repos.

Reply:
No objections from me.